CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 *2024-06-04*
 
+### Added
+
+- Add locations in the JSON5 parser error messages.
```

Comment:
This should include the PR number and your Github handle to credit you.

Reply:
Can you please check the format?

Reply:
Looks good now (we have a CI check for it), thanks.